### PR TITLE
testing/dumb-init: new aport

### PR DESCRIPTION
**Pull request by leprechau**
A minimal init system for Linux containers

https://github.com/Yelp/dumb-init

### CODE REVIEW

**Comment by jirutka** [comment]
Travis build failed, because dump-init uses bash in Makefile (for whatever reason), so you have to add `bash` to `makedepends`.

**Comment by leprechau** [comment]
Gotcha, thanks

File: testing/dumb-init/APKBUILD [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# Contributor: ena r&d <r&d@ena.com>
```
Comment (jirutka) [inline testing/dumb-init/APKBUILD]: Is this really a valid mail address?

Comment (leprechau) [inline testing/dumb-init/APKBUILD]: It is valid but it's a group list.  I can update those to point to me directly.

Comment (jirutka) [inline testing/dumb-init/APKBUILD]: That would be better.

File: testing/dumb-init/APKBUILD [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+builddir="$srcdir/$pkgname-$pkgver"
+build() {
+	cd "$builddir"
+	make || return 1
+}
+
+package() {
+	cd "$builddir"
+	install -cpsD -o root -g root -m 655 dumb-init "$pkgdir"/usr/bin/dumb-init || return 1
```
Comment (jirutka) [inline testing/dumb-init/APKBUILD]: Are you sure about 655? Shouldn’t it be 755?

Comment (leprechau) [inline testing/dumb-init/APKBUILD]: Yes, it should be 755

**Comment by jirutka** [comment]
Merged in 12ecd81bbe54bbd873c73c6bb81c41b124cadca5, thanks!

**Pull request closed without merging**